packages/manager/modules/hub/src/components/tile/tile.html

```diff
@@ -0,0 +1,27 @@
+<div class="oui-tile ovh-manager-hub-tile">
+    <div class="ovh-manager-hub-tile__header">
+        <p class="oui-heading_5 oui-tile__title">
+            <span data-ng-bind="::$ctrl.heading"></span>
+            <span
+                class="oui-chip ovh-manager-hub-tile__badge"
+                data-ng-bind="$ctrl.badgeText"
+            ></span>
+        </p>
+        <p class="hub-tile__button-container">
+            <a
+                class="oui-button oui-button_link hub-button"
```

Comment:
sometimes classes are prefixed https://github.com/ovh/manager/pull/2227/files#diff-702062823322e70d5a36cf25d9438f57R6, sometimes they're not. Any reason ?

Reply:
When I do that for the hub-button, for some reason, the `::before` style for the button gets overridden by the default css from oui-button_link. Not sure why this happens